packages/amplify-provider-awscloudformation/src/utils/generate-auth-nested-stack-params.ts

```diff
@@ -2,11 +2,11 @@ import { ProviderMeta, ProviderCreds, ProviderParameters } from './auth-types';
 
 export const generateAuthNestedStackParameters = (
   hostedUIProviderMeta: ProviderMeta[],
-  hostedUIProviderCreds: ProviderCreds[],
+  hostedUIProviderCreds2: ProviderCreds[],
```

Comment:
What does the 2 signify here?

Reply:
Oh that needs to be changed, it was for debugging